custom/icds_reports/tasks.py

```diff
@@ -174,10 +176,16 @@
                acks_late=True, queue='icds_aggregation_queue')
 def run_move_ucr_data_into_aggregation_tables_task():
     move_ucr_data_into_aggregation_tables.delay()
+    if toggles.PARALLEL_AGGREGATION.enabled(DASHBOARD_DOMAIN):
+        move_ucr_data_into_aggregation_tables.delay(force_citus=True)
 
 
 @serial_task('move-ucr-data-into-aggregate-tables', timeout=36 * 60 * 60, queue='icds_aggregation_queue')
-def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
+def move_ucr_data_into_aggregation_tables(date=None, intervals=2, force_citus=False):
+
+    if force_citus:
+        force_citus_engine()
```

Comment:
should this be unset afterwards? Might be safer to make a context manager that reset's the value on exit

Reply:
This thread local won't get applied to the sub-tasks that get executed. I think you'll still have to pass `force_citus` to the sub-tasks.